elisp-bootstrap/ghc-mod.el

```diff
@@ -0,0 +1,12 @@
+(defconst ghc-version "0")
+(defvar ghc-module-command "ghc-mod"
+"*The command name of \"ghc-mod\"")
+(defvar ghc-bootstrapped nil)
+
+(defun ghc-bootstrap ()
```

Comment:
Just prefix all symbols with ghc-mod instead of ghc I'd like to move away from the elisp package being called ghc, might as well start now.